scripts/beesd.in

```diff
@@ -18,44 +18,69 @@ readonly bees_bin=$(realpath @LIBEXEC_PREFIX@/bees)
 
 command -v "$bees_bin" &> /dev/null || ERRO "Missing 'bees' agent"
 
-## Parse args
+uuid_valid(){
+    if uuidparse -n -o VARIANT $1 | grep -i -q invalid; then
+        false
+    fi
+}
+
+help(){
+    echo "Usage: beesd [options] <btrfs_uuid>"
+    echo "- - -"
+    "$bees_bin" --help
+    exit 0
+}
+
+get_bees_supp_opts(){
+    "$bees_bin" |& grep -- '--..' | awk '{print $1 $2}' | sed 's/,/ /g'
```

Comment:
Why not
```
/usr/libexec/bees |& awk '/--../ {print $1 $2}' | sed 's/,/ /g'
```
`awk` is also `grep`.

Also, I think you should call `"$bees_bin"` with `--help` to not depend on implicit behavior.

Reply:
You can even get rid of `sed`:
```
"$bees_bin" --help |& awk '/--../ {print $1 " " $2}'
```

Reply:
LGTM